terraform/cron-loaders.tf

```diff
@@ -36,10 +36,14 @@ locals {
     hyvee          = { schedule = "cron(8/10 * * * ? *)" }
     heb            = { schedule = "cron(1/10 * * * ? *)" }
     cdcApi         = { schedule = "cron(0 0,12 * * ? *)" }
-    riteAidScraper = { schedule = "cron(0/10 * * * ? *)" }
+    riteAidScraper = {
+      schedule = "cron(0/10 * * * ? *)"
+      # FIXME: a bunch of states are broken in the scraper right now. Remove
+      # this narrow list when the unlisted states are fixed.
+      options = ["--states", "CT,DE,MA,NH,NJ,NV,OR,VT"]
+    }
     riteAidApi = {
       schedule = "cron(0/30 * * * ? *)"
-      options  = ["--states", "CA,CT,DE,ID,MA,MD,MI,NH,NJ,NV,NY,OH,OR,PA,VA,VT,WA"]
```

Comment:
This allow list of states was only used because the public API stopped supporting Colorado a while back, before it was clear those stores were permanently closed. Since there are definitely closed now, I removed them from the list of default states for Rite Aid, and this list is no longer needed.

(But now we need a list for the scraper, which uses the private API and is broken for some states. 🙃)